### ISSUE TITLE
Work out how/where we're going to host the Calico docs

### ISSUE THREAD

**Issue by eepyaich** [projectcalico/calico#8]
The format of the docs in the repo is such that they can be built/hosted by Github pages, if that's what we want.  

As it stands, the docs in the gh-master branch of this repo are hosted on tigera.github.io/calico-docs, but we'll presumably want to put the docs in a projectcalico repo, which would move their default location (e.g. projectcalico.github.io/calico).

Github pages supports custom domains (https://help.github.com/articles/using-a-custom-domain-with-github-pages/), so it should be possible to actually have them hosted by github but located somewhere else (e.g. replace the current docs.projectcalico.org).

Basically, we should be able to do what we want - just needs someone to make a decision on what that is.

**Comment by djosborne** [comment]
- Docs are now hosted on http://docs.projectcalico.org
- Served as a org project page from master branch of this repo (projectcalico/calico)
